### PR TITLE
Scotland_5 - Abdulrahman Salman - HTML/CSS - week_02

### PR DESCRIPTION
**Pull request by Abadi11**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +14,68 @@
...
+
+        <div class="introducing-karma">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <div class="learn-more"> <a href="#">Learn more</a> </div>
+        </div>
+    </header>
+
+    <main>
+        <div class="main-container">
```
Comment (bonboh) [inline index.html:40]:
```suggestion
    <main class="main-container">
```
We don't need the extra `div` inside `main`, so we can add the class to the `main` instead and delete the `div`

File: index.html [hunk 2]
```diff
@@ -14,6 +14,68 @@
...
+            <p>Everyone needs a little Karma</p>
```
Comment (bonboh) [inline index.html:41]:
```suggestion
            <h2>Everyone needs a little Karma</h2>
```
This feels like a heading so we could make it one

File: index.html [hunk 3]
```diff
@@ -14,6 +14,68 @@
...
+            <div class="image-container">
+                <div class="image-device">
+                    <img src="./img/icon-devices.svg" alt="Internet for all devices">
+                    <div class="desc"><p>Internet for all devices</p></div>
+                </div>
+                    
+                <div class="image-coffee">
+                    <img src="./img/icon-coffee.svg" alt="icon-coffee">
```
Comment (bonboh) [inline index.html:49]:
```suggestion
                    <img src="./img/icon-coffee.svg" alt="Coffee cup">
```
`alt` is a description of the image, so we can describe this one like "coffee cup". Same goes for the other images in this file

**Review by bonboh — COMMENTED** [review]
Good job @Abadi11! Getting it to look nice on a mobile would be a bonus too. I've left some comments and tips for you to read, but you don't have to do anything else 👍

File: css/style.css [hunk 4]
```diff
@@ -16,4 +16,114 @@ body {
...
  */
 
+/*Header Container*/
 
+.header-container {
+    position: relative;
+    width: 100%;
+    
+}
+/*Navigation Style*/
```
[the remark this answers is not in the document]
Comment (Abadi11) [reply css/style.css:26]: Great, thank you very much

File: css/style.css [hunk 5]
```diff
@@ -16,4 +16,114 @@ body {
...
+    width: 100%;
+
+}
+
+
+
+
+ul > li{
+    list-style-type: none;
+    display: inline-flex;
```
[the remark this answers is not in the document]
Comment (Abadi11) [reply css/style.css:42]: Got it, thank you very much

**Comment by Abadi11** [comment]
Thank you very much for your helping. I adjusted everything you noticed. Thanks

**Pull request closed without merging**